listenbrainz/webserver/views/art_api.py

```diff
@@ -225,7 +225,7 @@ def cover_art_custom_stats(custom_name, user_name, time_range, image_size):
 
 def _cover_art_yim_stats(user_name, stats):
     """ Create the SVG using YIM statistics for the given year. """
-    match stats["day_of_week"]:
+    match stats.get("day_of_week"):
```

Comment:
this means that if there are no items for `day_of_week` then it'll return `"I CRANKED TUNES ON <tspan class="user-stat">None</tspan>"`, is that what we want?

Reply:
Hmm, good point. Let me think about it.

Reply:
Fixed with return a 204 No Content in that case too.